compose/compose-file/index.md

```diff
@@ -44,7 +40,115 @@ tutorial.
 
 - [Adding a new service and redeploying](/get-started/part5.md#adding-a-new-service-and-redeploying)
 
-Another example of a Compose file is included in the Docker Labs topic, [Deploying an app to a Swarm](https://github.com/docker/labs/blob/master/beginner/chapters/votingapp.md).
+Another good example is the Compose file for the voting app sample used in the
+[Docker for Beginners lab](https://github.com/docker/labs/tree/master/beginner/)
+topic on [Deploying an app to a
+Swarm](https://github.com/docker/labs/blob/master/beginner/chapters/votingapp.md).
+Click to show/hide the example Compose file below.
+
+<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
+
+  <div class="panel panel-default">
+    <div class="panel-heading" role="tab" id="headingThree">
+      <h5 class="panel-title" id="collapsible-group-item-3"> <a class="" role="button" data-toggle="collapse" data-parent="#accordion" data-target="#collapseThree" aria-expanded="true" aria-controls="collapseThree"> Example Compose file version 3 </a> </h5>
+    </div>
+    <div id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThree" aria-expanded="true">
```

Comment:
@joaofnfernandes _great_ suggestion, have a look now, should be good. (FYI, I'm going to look into the possibility of being able to pull this file from the voting app sample on GitHub in the long term.)